hazelcast/src/test/java/com/hazelcast/test/mocknetwork/MockJoiner.java

```diff
@@ -149,12 +157,24 @@ private Node findAliveNode() {
                 continue;
             }
 
+            if (shouldIsolateClusters && isMemberOfIsolatedCluster(foundNode)) {
+                logger.fine("Node for " + address + " is outside of the join range and should not be joined.");
```

Comment:
I'm thinking if this should be a warn instead if the two members' configured ports (`config.network.port`) are the same.

Reply:
Yes, feel free to change it to a clearer message, feels like if we start getting some test failures because of this, it would be good if it was printed in the default logs + it's only for tests anyway.